backend-project/config/settings/base.py

```diff
@@ -146,6 +153,20 @@
     "PAGE_SIZE": 20,
 }
 
+AUTHENTICATION_BACKENDS = (
+    "social_core.backends.google.GoogleOAuth2",
+    "rest_framework_social_oauth2.backends.DjangoOAuth2",
+    "django.contrib.auth.backends.ModelBackend",
+)
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = env("SOCIAL_AUTH_GOOGLE_OAUTH2_KEY")
```

Comment:
Podoa mi się to, że jest to przekazywane w zmiennej środowiskowej, a nie umieszczone w bazie danych. :+1: